### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by mikerooni**

### CODE REVIEW

**Review by screret — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,44 +1,24 @@
...
+- Fixed compression recipes being generated for ignored materials
+- Fixed fluid locking
+- Fixed items being voided when Multiblock auto-building fails
+- Fixed missing drops for Laser Pipes and Optical Pipes
+- Fixed missing harder chest recipes
+- Fixed setting the dimension for Bedrock Ore Veins and Fluid Veins in KJS
+- Fixed energy transfer display in the Portable Scanner
+- Fixed parallel issues, multi smelter not overclocking
+- Fixed Overclocking display for ULV in recipe viewers
+- Fixed crash when parallel processing isn't possible
```
Comment (screret) [inline CHANGELOG.md]: hold up, this doesn't need to be mentioned

**Review by screret — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (screret) [reply CHANGELOG.md]: (the last highlighted one)

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,44 +1,23 @@
...
+Version: 1.2.2
 
 ### ADDITIONS:
-- Ported decorative blocks from 1.12
-- Added filtering to Output Hatches
-- Added several utility functions to KJS
-- Added more ore generator types (Classic & Cuboid)
-- Added partial NBT support for recipe inputs
-- Added Wood Cutter recipes
+- Ported Powder Barrels, Industrial TNT and Dynamite from 1.12
```
Comment (screret) [inline CHANGELOG.md:6]: Mention the armor?

**Review by screret — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (TheAlaskanPhysicist) [reply CHANGELOG.md:6]: Agreed. #1210 has been reviewed and approved

**Review by TheAlaskanPhysicist — CHANGES_REQUESTED** [review]
Change what Screret requested and we're good!

**Comment by screret** [comment]
also the rest of the stuff that came after this one

**Review by screret — APPROVED** [review]

**Review by TheAlaskanPhysicist — APPROVED** [review]
All good. LGTM

**Pull request merged**